stdlib/public/Darwin/Network/NWListener.swift

```diff
@@ -78,9 +78,9 @@ public final class NWListener: CustomDebugStringConvertible {
 			get {
 				let descriptor = nw_advertise_descriptor_create_bonjour_service(name, type, domain)
 				if let txtRecord = txtRecord {
-					txtRecord.withUnsafeBytes({ (ptr: UnsafePointer<UInt8>) -> Void in
-						nw_advertise_descriptor_set_txt_record(descriptor!, ptr, txtRecord.count)
-					})
+					txtRecord.withUnsafeBytes { (ptr: UnsafeRawBufferPointer) -> Void in
+                        nw_advertise_descriptor_set_txt_record(descriptor!, ptr.baseAddress!, txtRecord.count)
```

Comment:
Nit: As was said before, we ought to get this corrected before we merge.

Reply:
The type annotations for the closure were only needed to call the deprecated variant, so they ought to be removed. With the buffer pointer-based entry points, we prefer to extract the count from the buffer pointer itself, not the original class instance:

```swift
txtRecord.withUnsafeBytes { buffer in
 nw_advertise_descriptor_set_txt_record(descriptor!, buffer.baseAddress!, buffer.count)
}
```

Reply:
fixed